module/zfs/spa_misc.c

```diff
@@ -527,6 +529,18 @@ spa_config_enter(spa_t *spa, int locks, const void *tag, krw_t rw)
 	ASSERT3U(wlocks_held, <=, locks);
 }
 
+void
+spa_config_enter(spa_t *spa, int locks, const void *tag, krw_t rw)
+{
+	spa_config_enter_impl(spa, locks, tag, rw, 0);
+}
+
+void
```

Comment:
Can you add a block comment above `spa_config_enter_mmp()` explaining the mmp thread is allowed to cut in front of outstanding write lock requests because mmp updates are time sensitive and failure to service them promptly will result in a suspended pool.

Reply:
Sure! No problem.